functional_tests.go

```diff
@@ -5554,8 +5554,8 @@ func testStorageClassMetadataPutObject() {
 		logError(testName, function, args, startTime, "", "PutObject failed", err)
 		return
 	}
-	if !reflect.DeepEqual(metadata, fetchMeta("srcObjectSSClass")) {
-		logError(testName, function, args, startTime, "", "Metadata match failed", err)
+	if reflect.DeepEqual(metadata, fetchMeta("srcObjectSSClass")) {
+		logError(testName, function, args, startTime, "", "Metadata match failed, STANDARD should not be a part of response metadata", err)
```

Comment:
Suggest wording this as ""Metadata match failed, STANDARD storage class should not be a part of response metadata".